From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Wed, 26 Aug 2015 13:24:36 +0100
Subject: Makefile.am EXTRA_DIST

---
 autocrap/Makefile.am | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/autocrap/Makefile.am b/autocrap/Makefile.am
index 2ee7bd9..6f09b9c 100644
--- a/autocrap/Makefile.am
+++ b/autocrap/Makefile.am
@@ -42,3 +42,5 @@ noinst_LIBRARIES+=	libscrypt_sse2.a
 libscrypt_sse2_a_SOURCES=	lib/crypto/crypto_scrypt_smix_sse2.c
 libscrypt_sse2_a_CFLAGS=`. ./cpusupport-config.h; echo $${CFLAGS_X86_SSE2}`
 
+EXTRA_DIST = BSDmakefile FORMAT lib/README libcperciva/COPYRIGHT	\
+ lib/crypto/crypto_scrypt-ref.c
