From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Mon, 9 Mar 2015 16:44:51 +0000
Subject: warnings

enable compiler warnings
---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index 2c88538..b5bb322 100644
--- a/configure.ac
+++ b/configure.ac
@@ -43,6 +43,8 @@ AC_SUBST([SCRYPTVER])
 
 AC_SYS_LARGEFILE
 
+AX_CFLAGS_WARN_ALL
+
 AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_FILES([Makefile])
 AC_OUTPUT
