From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Wed, 26 Aug 2015 13:34:18 +0100
Subject: compiler warnings

enable compiler warnings
---
 autocrap/configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/autocrap/configure.ac b/autocrap/configure.ac
index 6d6db11..a9119fe 100644
--- a/autocrap/configure.ac
+++ b/autocrap/configure.ac
@@ -38,6 +38,8 @@ AC_CHECK_FUNCS([mmap])
 
 AC_SYS_LARGEFILE
 
+AX_CFLAGS_WARN_ALL
+
 AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_FILES([Makefile])
 AC_OUTPUT
