From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Mon, 9 Mar 2015 19:18:19 +0000
Subject: Makefile.am EXTRA_DIST

---
 Makefile.am | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Makefile.am b/Makefile.am
index 6e24d64..8ed4fe9 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -36,3 +36,6 @@ noinst_LIBRARIES=	libcperciva_aesni.a
 libcperciva_aesni_a_SOURCES=	libcperciva/crypto/crypto_aes_aesni.c
 libcperciva_aesni_a_CFLAGS=	$(scrypt_CFLAGS)	\
 			`. cpusupport-config.h; echo $${CFLAGS_X86_AESNI}`
+
+EXTRA_DIST = BSDmakefile FORMAT lib/README libcperciva/COPYRIGHT	\
+ lib/crypto/crypto_scrypt-ref.c
